api/producer/swagger.yaml

```
@@ -310,9 +310,9 @@ paths:
             ]
           }
           ```
-        * `category` SHOULD indicate the broader class of the Document Type as agreed during [onboarding](#api-description__onboarding). e.g.
+        * `category` MUST indicate the broader class of the Document Type as agreed during [onboarding](#api-description__onboarding). e.g.
           ```
-          "type": {
+          "category": {
```

Actually 'category', unlike type, is supposed to be in list format (even though we are only allowing one)... (also I realise this wasn't mentioned when discussing the ticket, I should have caught it earlier sorry).

```suggestion
          "category": { [
            "coding": [
              {
                "system": "http://snomed.info/sct",
                "code": "1102421000000108",
                "display": "Observations"
              }
            ]
          }
         ]
```

---

Type gets validated when trying to create the core model in the interaction using the DocumentPointer from the model class. So if it has multiple types or codings it throws an error.